opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -907,6 +915,16 @@ def __init__(
         if shutdown_on_exit:
             self._atexit_handler = atexit.register(self.shutdown)
 
+        self.span_attribute_count_limit = Configuration().get(
```

Comment:
at this point, you've almost done all the work to make this programmatically configurable: just make each of these defaults customizable, or some combination there of.

But not an issue if you choose not to do so.